<ui version="4.0" >
 <class>QRL_LedsManager</class>
 <widget class="QDialog" name="QRL_LedsManager" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>303</width>
    <height>228</height>
   </rect>
  </property>
  <property name="minimumSize" >
   <size>
    <width>303</width>
    <height>228</height>
   </size>
  </property>
  <property name="maximumSize" >
   <size>
    <width>303</width>
    <height>228</height>
   </size>
  </property>
  <property name="windowTitle" >
   <string>Leds Manager</string>
  </property>
  <property name="windowIcon" >
   <iconset resource="qrtailab.qrc" >:/icons/led_icon.xpm</iconset>
  </property>
  <layout class="QGridLayout" >
   <item rowspan="2" row="0" column="0" >
    <widget class="QListWidget" name="ledListWidget" >
     <property name="baseSize" >
      <size>
       <width>100</width>
       <height>0</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="0" column="1" colspan="4" >
    <widget class="QTabWidget" name="tabWidget" >
     <property name="currentIndex" >
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab" >
      <attribute name="title" >
       <string>Led</string>
      </attribute>
      <layout class="QGridLayout" >
       <item rowspan="2" row="0" column="0" colspan="3" >
        <widget class="QCheckBox" name="showCheckBox" >
         <property name="text" >
          <string>Show/Hide</string>
         </property>
        </widget>
       </item>
       <item row="1" column="2" >
        <spacer>
         <property name="orientation" >
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" >
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="2" column="0" >
        <widget class="QLabel" name="colorLabel" >
         <property name="text" >
          <string>Led color</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1" colspan="2" >
        <spacer>
         <property name="orientation" >
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" >
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="3" column="0" >
        <widget class="QComboBox" name="ledColorComboBox" >
         <item>
          <property name="text" >
           <string>Red</string>
          </property>
         </item>
         <item>
          <property name="text" >
           <string>Green</string>
          </property>
         </item>
         <item>
          <property name="text" >
           <string>Blue</string>
          </property>
         </item>
         <item>
          <property name="text" >
           <string>Yellow</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="3" column="1" colspan="2" >
        <spacer>
         <property name="orientation" >
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" >
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="4" column="0" >
        <spacer>
         <property name="orientation" >
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" >
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item row="1" column="1" >
    <spacer>
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" >
      <size>
       <width>16</width>
       <height>27</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="2" >
    <widget class="QPushButton" name="helpButton" >
     <property name="text" >
      <string>Help</string>
     </property>
    </widget>
   </item>
   <item row="1" column="3" >
    <spacer>
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" >
      <size>
       <width>16</width>
       <height>27</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="4" >
    <widget class="QPushButton" name="closeButton" >
     <property name="text" >
      <string>close</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="qrtailab.qrc" />
 </resources>
 <connections>
  <connection>
   <sender>closeButton</sender>
   <signal>clicked()</signal>
   <receiver>QRL_LedsManager</receiver>
   <slot>hide()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>314</x>
     <y>244</y>
    </hint>
    <hint type="destinationlabel" >
     <x>381</x>
     <y>239</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
